src/Feature/Editor/EditorConfiguration.re

```diff
@@ -238,10 +271,18 @@ let fontFamily =
     ~vim=VimSettings.guifont,
     "editor.fontFamily",
     string,
+    // XXX: Might have to remove the regular part"
     ~default="JetBrainsMono-Regular.ttf",
   );
 let fontLigatures = setting("editor.fontLigatures", bool, ~default=true);
 let fontSize = setting("editor.fontSize", fontSize, ~default=14.);
+// XXX: Set the vim named argument?
```

Comment:
Good call here - I'm OK deferring this for now (It'll be a bit of extra work - we'd need to extend our parser to handle more intricate formats for the `guifont` setting).

Reply:
Ok, I'll skip this one for now.